api/src/auth/authRoutes.ts

```diff
@@ -465,6 +471,51 @@ export function addAuthRoutes(app: Router, socialProviders: AuthProvider[]) {
   });
 
   /**
+   * Logout - logging out is an optional client 'good will' process in which the
+   * client indicates they wish for the specific refresh token to be
+   * invalidated. This is just a normal POST operation.
```

Comment:
Did you mean POST or PUT? It might be PUT if you were updating the state of an auth URI but you're not really doing that, POST is probably appropriate.